Platform/CometlakeBoardPkg/CfgData/Template_CfgData.yaml

```diff
@@ -381,14 +381,20 @@ GPIO_TMPL: >
         condition    : $GPIO_CFG_DATA.GpioPinConfig1_$(1).GPIOSkip_$(1) == 0
 
         length       : 4b
-
+    - RxRaw_$(1) :
+        name         : RxRaw
+        type         : Combo
+        option       : 0x0:Default, 0x2:NoOverride, 0x3:RxDrive_1_internally

```

Comment:
1 for NoOverride ?

Reply:
Thanks for catching this. Updated in new patch.